apps/typegpu-docs/src/components/ExampleView.tsx

```diff
@@ -187,7 +185,7 @@ export function ExampleView({ example, isPlayground = false }: Props) {
           </div>
 
           {codeEditorShowing ? (
-            <div className="relative h-full">
+            <div className={'absolute z-20 md:relative h-full w-full'}>
```

Comment:
Tiny nitpick, unnecessary interpolation.

```suggestion
            <div className="absolute z-20 md:relative h-full w-full">
```